package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Project 2 Starter",
   "main": "server.js",
   "scripts": {
-    "start": "node server.js",
+    "start": "nodemon server.js",
```

Comment:
start script should use node. we can add a "watch" script that uses nodemon like in other class activities

Reply:
Fixed.

Reply:
Left is untouched for now.. I could not get the watch to work. Will fix in next PR